source/mainnet/_templates/index.html

```diff
@@ -24,23 +24,61 @@
+                    <li><a href="{{ pathto('net/desktop-wallet/send-gtu-single-desktop') }}">Make a transfer with the desktop wallet</a></li>
                 <h3>Participate in the network</h3>
                 <ul class="feature-box-list">
                     <li><a href="{{ pathto('net/guides/overview-baker-process') }}">Get an overview of the baker process</a></li>
                 </ul>
             </div>
         </div>
+        <div class="feature">
+            <div class="feature-box-testnet">
+                <a href="{{ pathto('en/testnet/net/index') }}">
+                    <div class="feature-box-header-testnet">
+                        <h1>Testnet</h1>
+                    </div>
+                </a>
+                <h3>Create an account</h3>
+                <ul class="feature-box-list">
+                    <li><a href="{{ pathto('en/testnet/net/installation/downloads') }}">Download a wallet</a></li>
+                    <li><a href="{{ pathto('en/testnet/net/desktop-wallet/create-identity-desktop') }}">Create your first account using <br> the Desktop Wallet</a></li>
+                    <li><a href="{{ pathto('en/testnet/net/mobile-wallet/get-started') }}">Create your first account using <br> the Mobile Wallet</a></li>
+                    <li><a href="{{ pathto('en/testnet/net/desktop-wallet/install-ledger-app') }}">Setup a Ledger Nano S for <br> the Desktop Wallet</a></li>
+                </ul>
+                <h3>Make a transfer</h3>
+                <ul class="feature-box-list">
+                    <li><a href="{{ pathto('en/testnet/net/mobile-wallet/accounts-transactions') }}">Make a transfer with the mobile wallet</a></li>
+                    <li><a href="{{ pathto('en/testnet/net/desktop-wallet/send-gtu-single-desktop') }}">Make a transfer with the desktop wallet</a></li>
+                </ul>
+                <h3>Participate in the network</h3>
+                <ul class="feature-box-list">
+                    <li><a href="{{ pathto('en/testnet/net/guides/run-node-windows') }}">Run a node on Windows</a></li>
+                    <li><a href="{{ pathto('en/testnet/net/guides/run-node-macos') }}">Run a node on Mac</a></li>
+                    <li><a href="{{ pathto('en/testnet/net/guides/guides/run-node') }}">Run a node on Linux</a></li>
+                    <li><a href="{{ pathto('en/testnet/net/guides/overview-baker-process') }}">Get an overview of the baker process</a></li>
+                </ul>
+                <h3>Smart Contracts</h3>
+                <ul class="feature-box-list">
+                    <li><a href="{{ pathto('en/testnet/smart-contracts/general/introduction') }}">Introduction to smart contracts</a></li>
+                    <li><a href="{{ pathto('en/testnet/smart-contracts/guides/setup-tools') }}">Install the necessary tools</a></li>
+                    <li><a href="{{ pathto('en/testnet/smart-contracts/tutorials/piggy-bank/index') }}">Tutorial: A piggy bank smart contract</a></li>
```

Comment:
To keep the parallel structure the same, we could change the wording a bit. For example:
Get started with smart contracts
Install...
Learn with the piggy bank smart contract tutorial